functions/Move-DbaDbFile.ps1

```diff
@@ -0,0 +1,366 @@
+function Move-DbaDbFile {
+    <#
+    .SYNOPSIS
+        Moves database files from one location to another.
+
+    .DESCRIPTION
+        Moves database files from one location to another.
```

Comment:
I would recommend defining location. 

SQL Server 2016 brought support for storing data files on Azure Blob storage service...are we going to support that in this command? If not I would recommend layout what location means.

Example:

> Move database files from one local drive or folder to another.

Reply:
Which to even move it to Azure Blob can only be done via backup/restore, but to prevent confusion or questions I'd say local drives/folders.

Reply:
I agree. I updated it.
Thank you, Shawn.